tailwind.config.js

```diff
@@ -1,18 +1,46 @@
-/* eslint-disable global-require */
+/* eslint-disable import/no-extraneous-dependencies, global-require */
+const defaultTheme = require('tailwindcss/defaultTheme');
+
 module.exports = {
   mode: 'jit',
   purge: ['./src/**/*.{js,jsx,ts,tsx}'],
   darkMode: false,
+  corePlugins: {
+    container: false,
+  },
   theme: {
     screens: {
-      '2xl': { max: '1535px' },
+      // TODO: Uncomment and change the value of 2xl breakpoint if it's needed, otherwise just delete it
+      // '2xl': { max: '1535px' },
       xl: { max: '1279px' },
       lg: { max: '1023px' },
       md: { max: '767px' },
       sm: { max: '639px' },
       xs: { max: '413px' },
     },
-    extend: {},
+    extend: {
```

Comment:
Since we don't use any default colors and fonts, instead of defining them in `extend` section we can put it within `theme`

https://tailwindcss.com/docs/customizing-colors#curating-colors